src/components/AuthInputField/index.tsx

```diff
@@ -11,8 +11,8 @@ type AuthInputFieldProps = {
 
 const AuthInputField = ({ type, name, id, className, label, dataTestID }: AuthInputFieldProps): JSX.Element => {
   return (
-    <div className="form-input-group">
-      <label className="form-input-group__label" htmlFor={id}>
+    <div className="form-input-group-auth">
```

Comment:
Why was this changed? IMO, `form-input-group` was nice. If you think you need to use a modified `form-input-group` then we could use modifier for those. i.e. `form-input-group form-input-group--auth`. As for this IC, as there are no more such forms in the design, we could just use `form-input-group`. What do you think?

Reply:
Agree with you if we don't have any other form we can just use what you suggest. Fixed in abcf929ecfc7dcb262763156a579f7fbc4722f19